plugins/check_ups.c

```diff
@@ -615,6 +620,8 @@ print_help (void)
   printf ("    %s\n", _("Name of UPS"));
   printf (" %s\n", "-T, --temperature");
   printf ("    %s\n", _("Output of temperatures in Celsius"));
+  printf (" %s\n", "-e, --extended-units");
+  printf ("    %s\n", _("Allow nonstandard units in performance data (used for voltage an temperatures)."));
```

Comment:
@Madlohe an -> and; "." at the end is probably not necessary (not used for other options).

Reply:
Good catch, I've committed the changes to maint.